libweston/backend-rdp/rdprail.c

```diff
@@ -363,10 +371,9 @@ rail_client_SnapArrange_callback(bool freeOnly, void *arg)
 				snapArrangeRect.y,
 				snapArrangeRect.width,
 				snapArrangeRect.height);
+			rail_state->forceUpdateWindowState = true;
+			rdp_rail_schedule_update_window(NULL, (void*)surface);

```

Comment:
for upstream we'd use `(void *)` instead of `(void*)`

Reply:
Changed.

Reply:
oh, yes, dropping the casts entirely was better still. thanks!